spec/DemographicDataForTouchstone.schema.json

```diff
@@ -1,10 +1,10 @@
 {
-    "id": "DemographicDatasetForTouchstone",
+    "id": "DemographicDataForTouchstone",
     "type": "object",
```

Comment:
Renaming as `demographic_dataset` is now the name of a database table, whereas this maps onto the `demographic_statistic` table.